tools/benchmarks/llm_eval_harness/meta_eval_reproduce/meta_eval.py

```diff
@@ -0,0 +1,194 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
ideally would be great if we could replace this script with a preprocessing script and then use harness CLI to run the local config folder.

Reply:
Now this python script has been renamed to prepare_meta_eval.py, which will prepare and print out a lm_eval command for people to run.